Makefile

```diff
@@ -20,7 +20,7 @@ test-in-container: image-test
 	docker run -it -e GITHUB_USER=${GITHUB_USER} -e GITHUB_TOKEN=${GITHUB_TOKEN} $(TEST_IMAGE_NAME)
 
 test:
-	PYTHONPATH=$(CURDIR) pytest -v
+	PYTHONPATH=$(CURDIR) pytest-3 -v
```

Comment:
Not going to fly. In [Dockerfile.test](https://github.com/user-cont/release-bot/blob/master/Dockerfile.test#L16) we install pytest with pip3, so `pytest --version` should return 3.x.y.
`pytest-3` is a name of binary in `python3-pytest` rpm, which we don't use

But please add the `--showlocals`

Reply:
Figured it out finally locally. Will add.